src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -127,8 +127,11 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
       let caseEventData: CaseEventData = this.formValueService.sanitise(pageFormFields) as CaseEventData;
       caseEventData.event_token = this.eventTrigger.event_token;
       caseEventData.ignore_warning = this.ignoreWarning;
-      caseEventData.event_data = this.editForm.value.data;
-
+      let event_data = this.editForm.value.data
```

Comment:
can you move this code into a private method with a good descriptive name please? it's not clear why we are including this code

Reply:
Done!